astroquery/solarsystem/pds/core.py

```diff
@@ -0,0 +1,493 @@
+                    format="fixed_width",
+                    col_starts=(5, 18, 29),
+                    col_ends=(18, 29, 36),
+                    names=("ring", "pericenter", "ascending node"),
+                )
+
+                units_list = [None, u.deg, u.deg]
+                ringtable = table.QTable(ringtable, units=units_list)
+
+            # Saturn F-ring data
+            elif group.startswith("F Ring"):
+                lines = group.split("\n")
+                for line in lines:
+                    l = line.split(":")
+                    if "F Ring pericenter" in l[0]:
+                        peri = float(re.sub("[a-zA-Z]+", "", l[1]).strip(", \n()"))
+                    elif "F Ring ascending node" in l[0]:
+                        ascn = float(l[1].strip(", \n"))
+                ringtable = table.QTable(
+                    [["F"], [peri], [ascn]],
+                    names=("ring", "pericenter", "ascending node"),
+                    units=(None, u.deg, u.deg),
+                )
+
+            # Neptune ring arcs data
+            elif group.startswith("Courage"):
+                lines = group.split("\n")
+                for i in range(len(lines)):
+                    l = lines[i].split(":")
+                    ring = l[0].split("longitude")[0].strip(", \n")
+                    [min_angle, max_angle] = [
+                        float(s.strip(", \n"))
+                        for s in re.sub("[a-zA-Z]+", "", l[1]).strip(", \n()").split()
+                    ]
+                    if i == 0:
+                        ringtable = table.QTable(
+                            [[ring], [min_angle], [max_angle]],
+                            names=("ring", "min_angle", "max_angle"),
+                            units=(None, u.deg, u.deg),
+                        )
+                    else:
+                        ringtable.add_row([ring, min_angle*u.deg, max_angle*u.deg])
+
+            else:
+                pass
+
+        # do some cleanup from the parsing job
+        if ringtable is not None:
+            ringtable.add_index("ring")
+
+        bodytable = table.join(bodytable, bodytable2)  # concatenate minor body table
+        bodytable.add_index("Body")
+
+        systemtable["obs_time"] = Time(
+            obs_time, format="iso", scale="utc"
+        )  # add obs time to systemtable
+
+        return systemtable, bodytable, ringtable
+
+    def _parse_result(self, response, verbose=None):
+        """
+        Routine for managing parser calls
+
+        Parameters
+        ----------
+        self :  RingNodeClass instance
+        response : string
+            raw response from server
+
+        Returns
+        -------
+        systemtable : dict
+        bodytable : `astropy.Table`
+        ringtable : `astropy.Table`
+        """
+        self.last_response = response
+        try:
+            systemtable, bodytable, ringtable = self._parse_ringnode(response.text)
+        except Exception as ex:
```

Comment:
again, no need for the exception. If you expect some errors from the server, always can add a `response.raise_for_status()` after calling `self._request`